lib/bolt/transport/docker/connection.rb

```diff
@@ -51,6 +51,7 @@ def execute(*command, options)
           command_options = []
           # Need to be interactive if redirecting STDIN
           command_options << '--interactive' unless options[:stdin].nil?
+          command_options << '--tty' if options[:tty]
```

Comment:
@glennsarti you want to validate this on Windows?

IIRC this won't work

Reply:
Nope.  `--tty` works fine, and for me is needed for cmd.exe

```
C:\Source> docker run --interactive ruby:2.5 bash

bash: line 1: $'\r': command not found

bash: line 2: $'\r': command not found

bash: line 3: $'\r': command not found
asdads
bash: line 4: $'asdads\r': command not found


C:\Source> docker run --interactive --tty ruby:2.5 bash
root@b528a0cd9c55:/# Hello
bash: Hello: command not found
root@b528a0cd9c55:/# ^C
root@b528a0cd9c55:/# exit
exit
C:\Source>
```

Reply:
The `\r` is me pressing enter